File: lc1346.py
from typing import List


class Solution:
    def checkIfExist(self, arr: List[int]) -> bool:
        seen = set()

        for a in arr:
            if a * 2 in seen or a % 2 == 0 and a // 2 in seen:
                return True
            seen.add(a)
        return False
print(Solution.checkIfExist(0,[2,3]))

File: lc1346.java
public class lc1346 {
    public static void main(String[] args) {
        int arr[]={-10,12,-20,-8,15};
        boolean ans=false;
        for (int i=0;i<arr.length;i++){
            for (int j=0;j<arr.length;j++){
                if (arr[j]==arr[i]*2 || i!=j){
                    ans=true;
                    break;
                }
            }
        }
        System.out.println(ans);
    }
}
